CHANGELOG.md

```diff
@@ -1,5 +1,27 @@
 # Change Log
 
+
+## Version 1.0.0
+* Release date: September 24, 2018
+* Release status: General Availability
+
+## What's new in this version
+* Announcing the SQL Server 2019 Preview extension.
+  * Support for SQL Server 2019 preview features including big data cluster support.
+  * Azure Data Studio Notebooks
+  * The Azure Resource Explorer viewlets you browse data-related endpoints for your Azure accounts and create connections to them in Object Explorer. In this release Azure SQL Databases and servers are supported.
+  * SQL Server Polybase Create External Table Wizard
+* Query Results Grid performance and UX improvements for large number of result sets.
+* Visual Studio Code source code refresh from 1.23 to 1.26.1 with Grid Layout and Improved Settings Editor (preview).
+* Accessibility improvements for screen reader, keyboard navigation and high-contrast.
+* Added Connection name option to provide an alternative display name in the Servers view-let.
```

Comment:
I'll update view-let to "viewlet" for consistency, though let's discuss if we want to use that term in user facing docs.